File: java/vetores/MaiorPosicao.java

import entities.ExercicioMaiorPosicao;

import java.util.Locale;
import java.util.Scanner;

public class MaiorPosicao {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int i, N, posicao;
        double maior, valor;

        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        ExercicioMaiorPosicao[] num = new ExercicioMaiorPosicao[N];

        for (i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            valor = sc.nextDouble();
            num[i] = new ExercicioMaiorPosicao(valor);
        }

        posicao = 0;
        maior = num[0].getNum();

        for (i = 0; i < N; i++) {
            if (num[i].getNum() > maior) {
                maior = num[i].getNum();
                posicao = i;
            }
        }

        System.out.printf("MAIOR VALOR = %.1f %n", maior);
        System.out.printf("POSICAO DO MAIOR VALOR = %d", posicao);
    }
}

File: js/vetores/maiorPosicao.js
const readline = require('readline-sync');

const X = Number(readline.question("Quantos numeros voce vai digitar? "));
const numeros = [];

for (let i = 0; i < X; i++) {
    const num = Number(readline.question("Digite um numero: "));
    numeros.push(num);
}

let maior = numeros[0];
let posicao = 0;

for (let i = 0; i < X; i++) {
    if (numeros[i] > maior) {
        maior = numeros[i];
        posicao = i;
    }
}

console.log(`MAIOR VALOR = ${maior}`);
console.log(`POSICAO DO MAIOR VALOR = ${posicao}`);